### PR TITLE
Input fixes

### PR DESCRIPTION
**Pull request by rnowm**
@basiclines visual changes in "input areas" and copied search form from "input areas" to "header" BB

### CODE REVIEW

File: widgets/headers.css [hunk 1]
```diff
@@ -43,20 +43,20 @@ section[role="region"] > header:first-child form {
   display: block;
   overflow: hidden;
   position: relative;
-  padding: 1.1rem;
+  padding: 0.9rem 0.9rem 0 0.2rem ;
```
Comment (basiclines) [inline widgets/headers.css]: For a consistent alignment, i think that the values should be `1rem 1rem 0 0.5rem` If not seems to be a little bit misaligned :S

File: widgets/headers.css [hunk 2]
```diff
@@ -65,7 +65,7 @@ section[role="region"] > header:first-child form button[type="reset"] {
   overflow: hidden;
   position: absolute;
   right: 1rem;
-  top: 0;
+  top: 0.9rem;
```
Comment (basiclines) [inline widgets/headers.css]: The same applies to this, it should be `1rem` imho

File: widgets/headers.css [hunk 3]
```diff
@@ -286,8 +286,111 @@ section[role="region"] > header + form.search input[type="text"] {
 
 section[role="region"] > header + form.search input[type="text"]:-moz-placeholder {
   color: #777;
+}*/
+
+form[role="search"] {
```
Comment (basiclines) [inline widgets/headers.css]: About the `search` as we have this component right now in `input_areas we do not need to duplicate it. So i suggest to remove all the code related to this search form.

File: widgets/headers/index.html [hunk 4]
```diff
@@ -55,8 +55,12 @@
       </menu>
       <h1>Inbox <em>(2)</em></h1>
     </header>
-    <form class="search">
-      <input type="text" placeholder="Search">
+    <form role="search" class="full">
+      <button type="submit">Send</button>
```
Comment (basiclines) [inline widgets/headers/index.html]: The same as above :D

**Comment by rnowm** [comment]
done!

**Comment by basiclines** [comment]
With this changes i feel comfortable merging this

**Pull request merged**